File: DSA/Arrays/count-of-elements/javascript/carryForward.js
export const countOfElements = (A) => {
        let max = A[0];
        let count = 0;
        if(A.length <=1) {
            return 0
        }
        for(let i = 1; i < A.length; i++){
            if (max < A[i]) {
                max = A[i] 
                count = 1
            } else if(max === A[i] ) {count += 1}
        }
        return A.length - count
	}

File: DSA/Arrays/carry-forward/count-of-elements/python/carryForward.py
def countOfElements(self, A):
        max = A[0]
        count = 1
        length = len(A) 
        for i in range(1, length):
            if max < A[i]:
                max = A[i]
                count = 1
            elif max == A[i]:
                count += 1
        return length - count